StripeCameraCore/StripeCameraCore/Source/Coordinators/CameraSession.swift

```diff
@@ -441,6 +441,7 @@ extension CameraSession {
         guard let captureDevice = captureDevices.devices.first else {
             throw CameraSessionError.captureDeviceNotFound
         }
+        print("BGLM - captureDevice: \(captureDevice)")
```

Comment:
did you mean to include this print statement?

Reply:
removed!